src/tex/appendix.tex

```diff
@@ -447,8 +447,19 @@ \subsection{Impact of sampling temperature}
 \end{figure}
 
 
+\clearpage
+\subsection{Summary of trends and recommendations}
+
+\oone consistently leads across both \ChemBench corpus and \chembenchmini.  \ClaudeThreeFiveSonnet, \GPTFourO, \LlamaThreeOneFourZeroFiveBInstruct form the next tier of strong performers. These models show robust performance across all skills and difficulty levels. 
+
+Larger models generally perform better (\LlamaThreeOneFourZeroFiveBInstruct > \LlamaThreeOneSeventyBInstruct > \LlamaThreeOneEightBInstruct). However, smaller but well-designed models can compete with larger models. For example, \GemmaTwoNineBIt has an overall accuracy of 0.48 on \ChemBench corpus, which is only 10\% less accurate than the much larger \LlamaThreeOneFourZeroFiveBInstruct model.
+
+One can observe a clear progression of performance across model families (e.g., \ClaudeThreeFiveSonnet >  \ClaudeThree > \ClaudeTwo or \GPTFourO > \GPTFour > \GPTThreeFiveTurboZeroT). Newer versions consistently outperform their predecessors, indicating improvements in scientific understanding of these model families.
+Scores on knowledge-intensive tasks are typically lower than those on calculation and reasoning-intensive questions. Performance on reasoning-intensive questions in \chembench tends to be the highest.
 
+Based on these findings, some scope for improving these models could be to focus on enhancing knowledge-based training, possibly through improved pre-training on chemistry-specific texts or with chemistry knowledge-base integration. However, this type of knowledge should be integrated with sound machine learning approaches. For example, \GalacticaOneTwentyB showed the lowest perforamnce in our study, even though it was specifically trained on scientific text. This suggests that the knowledge embedded in papers and textbooks has not been fully leveraged.
 
+Models show the strongest performance in reasoning tasks on the \chembench corpus, which suggests that current training approaches are good at developing logical/reasoning capabilities for basic problem-solving of university-level exams. However, performance drops significantly for advanced tasks, suggesting that more advanced chemistry problems with complex, multi-step solutions should be included in training or finetuning.
```

Comment:
```suggestion
Models show the strongest performance in reasoning tasks on the \chembench corpus, which suggests that current training approaches are good at developing logical/reasoning capabilities for basic problem-solving of university-level exams. However, performance drops significantly for advanced tasks, suggesting that more advanced chemistry problems with complex, multi-step solutions should be included in training or finetuning.
```

perhaps we can make this more effective by specifying an advanced chemistry problem (the ion lottery might be a naiive one)